<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqWindowCube</class>
 <widget class="QMainWindow" name="pqWindowCube">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout" name="_2">
    <item>
     <layout class="QHBoxLayout" name="_3">
      <item>
       <widget class="QGroupBox" name="thresholdGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Threshold</string>
        </property>
        <layout class="QVBoxLayout" name="_4">
         <item>
          <widget class="QSlider" name="thresholdSlider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="thresholdText"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="cuttingPlaneGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Cutting plane</string>
        </property>
        <layout class="QVBoxLayout" name="_5">
         <item>
          <widget class="QSlider" name="sliceSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="_19">
           <item>
            <widget class="QSpinBox" name="sliceSpinBox">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="velocityText">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="contourGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Contours</string>
        </property>
        <layout class="QHBoxLayout" name="_6">
         <item>
          <layout class="QVBoxLayout" name="_7">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Level</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="levelText">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>15</string>
             </property>
             <property name="maxLength">
              <number>4</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="_8">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Lower bound</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lowerBoundText">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="_9">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Upper bound</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="upperBoundText">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="contourCheckBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="valueGroupBox">
        <property name="title">
         <string>Value</string>
        </property>
        <layout class="QHBoxLayout" name="_10">
         <item>
          <widget class="QGroupBox" name="cubeGroupBox">
           <property name="title">
            <string>Datacube</string>
           </property>
           <layout class="QHBoxLayout" name="_11">
            <item>
             <layout class="QVBoxLayout" name="_13">
              <item>
               <widget class="QLabel" name="label">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Min</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="minCubeText">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="_14">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Max</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="maxCubeText">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="_15">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>RMS</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="rmsCubeText">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="sliceGroupBox">
           <property name="title">
            <string>Slice</string>
           </property>
           <layout class="QHBoxLayout" name="_12">
            <item>
             <layout class="QVBoxLayout" name="_16">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Min</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="minSliceText">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="_17">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Max</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="maxSliceText">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="PVLayout"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="_18">
      <item>
       <widget class="QVTKOpenGLNativeWidget" name="qVtkCube" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QVTKOpenGLNativeWidget" name="qVtkSlice" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionFront"/>
    <addaction name="actionBack"/>
    <addaction name="actionTop"/>
    <addaction name="actionRight"/>
    <addaction name="actionBottom"/>
    <addaction name="actionLeft"/>
   </widget>
   <widget class="QMenu" name="menuMoment">
    <property name="title">
     <string>Moment</string>
    </property>
    <addaction name="actionCalculate_order_0"/>
    <addaction name="actionCalculate_order_1"/>
   </widget>
   <widget class="QMenu" name="menuRendering">
    <property name="title">
     <string>Rendering</string>
    </property>
    <widget class="QMenu" name="menuVolume">
     <property name="title">
      <string>Volume</string>
     </property>
     <widget class="QMenu" name="menuOpacity">
      <property name="title">
       <string>Opacity</string>
      </property>
      <addaction name="action0"/>
      <addaction name="action25"/>
      <addaction name="action50"/>
      <addaction name="action100"/>
     </widget>
     <addaction name="actionVolGenerate"/>
     <addaction name="menuOpacity"/>
    </widget>
    <widget class="QMenu" name="menuSlice">
     <property name="title">
      <string>Slice</string>
     </property>
     <widget class="QMenu" name="menuColor_Map">
      <property name="title">
       <string>Color Map</string>
      </property>
      <addaction name="separator"/>
     </widget>
     <addaction name="menuColor_Map"/>
    </widget>
    <addaction name="menuVolume"/>
    <addaction name="menuSlice"/>
   </widget>
   <addaction name="menuCamera"/>
   <addaction name="menuMoment"/>
   <addaction name="menuRendering"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFront">
   <property name="icon">
    <iconset resource="../visivo.qrc">
     <normaloff>:/icons/PIC_FRONT.bmp</normaloff>:/icons/PIC_FRONT.bmp</iconset>
   </property>
   <property name="text">
    <string>Front</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="../visivo.qrc">
     <normaloff>:/icons/PIC_BACK.bmp</normaloff>:/icons/PIC_BACK.bmp</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="icon">
    <iconset resource="../visivo.qrc">
     <normaloff>:/icons/PIC_TOP.bmp</normaloff>:/icons/PIC_TOP.bmp</iconset>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="icon">
    <iconset resource="../visivo.qrc">
     <normaloff>:/icons/PIC_RIGHT.bmp</normaloff>:/icons/PIC_RIGHT.bmp</iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionBottom">
   <property name="icon">
    <iconset resource="../visivo.qrc">
     <normaloff>:/icons/PIC_BOTTOM.bmp</normaloff>:/icons/PIC_BOTTOM.bmp</iconset>
   </property>
   <property name="text">
    <string>Bottom</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="icon">
    <iconset resource="../visivo.qrc">
     <normaloff>:/icons/PIC_LEFT.bmp</normaloff>:/icons/PIC_LEFT.bmp</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionCalculate_order_0">
   <property name="text">
    <string>Calculate order 0</string>
   </property>
  </action>
  <action name="actionCalculate_order_1">
   <property name="text">
    <string>Calculate order 1</string>
   </property>
  </action>
  <action name="action0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
  </action>
  <action name="action25">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>25%</string>
   </property>
  </action>
  <action name="action50">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>50%</string>
   </property>
  </action>
  <action name="action100">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </action>
  <action name="actionVolGenerate">
   <property name="text">
    <string>Generate</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLNativeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../visivo.qrc"/>
 </resources>
 <connections/>
</ui>
